packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -99,11 +99,13 @@ small {
 }
 
 kbd {
+  display: inline-block;
```

Comment:
I don't mind sneaking these in, just wanna make sure we account for the change in CHANGELOG